backend/wmg/data/query.py

```diff
@@ -0,0 +1,41 @@
+from typing import List
+
+from pandas import DataFrame
+from pydantic import BaseModel, Field
+from tiledb import Array
+
+
+class WmgQueryCriteria(BaseModel):
+    gene_term_ids: list[str] = Field(default=[], unique_items=True, min_items=0)
+    organism_term_id: str
+    tissue_term_ids: list[str] = Field(min_items=1, unique_items=True)
```

Comment:
It's correct since organism and tissue are the two required params.  added comments to call this out explicitly (pedantic doesn't provide an explicit "required" attribute; it's assumed...)